pykern/xlsx.py

```diff
@@ -254,7 +257,8 @@ def _compile_expr(self, expr):
                 value=v,
                 is_decimal=True,
             )
-        if e[0].isalpha():
+        # TODO(robnagler) document that links must begin with alnum
+        if e[0][0].isalnum():
```

Comment:
A little confused about what e is. Would it be possible/beneficial to use PKDict with e instead of two dimensional arrays?

Reply:
`e` is the expression. `e[0]` is the first element, which has to be a string. `e[0][0]` is the first character of the string.